src/software/ai/evaluation/intercept.cpp

```diff
@@ -86,3 +87,41 @@ std::optional<std::pair<Point, Duration>> findBestInterceptForBall(const Ball &b
 
     return std::make_pair(best_ball_intercept_pos, time_to_ball_pos);
 }
+
+Point findInterceptionPoint(const Robot &robot, const Ball &ball, const Field &field)
+{
+    static constexpr double BALL_MOVING_SLOW_SPEED_THRESHOLD   = 0.3;
+    static constexpr double INTERCEPT_POSITION_SEARCH_INTERVAL = 0.1;
+
```

Comment:
let's move to the top of this file so it's easier to find

Reply:
I'll make these params